Author: Jonathan Dowland <jmtd@debian.org>
Date:   Mon, 19 Nov 2018 16:40:57 +0000

    Replace default preferences values with sensible defaults
    
    Sensible defaults for Debian systems

diff --git a/src/main/java/org/redmars/wadc/WadCPrefs.java b/src/main/java/org/redmars/wadc/WadCPrefs.java
index 5ee1ad5..e587d2d 100644
--- a/src/main/java/org/redmars/wadc/WadCPrefs.java
+++ b/src/main/java/org/redmars/wadc/WadCPrefs.java
@@ -66,10 +66,10 @@ public class WadCPrefs {
   public WadCPrefs() {
       prefs = Preferences.userNodeForPackage(WadCPrefs.class);
 
-      defs.put("bspcmd",   "c:\\doom2\\bsp");
-      defs.put("doomexe",  "c:\\doom\\zdoom.exe");
+      defs.put("bspcmd",   "bsp");
+      defs.put("doomexe",  "doom");
       defs.put("doomargs", "-warp 1 -file");
-      defs.put("iwad",     "c:\\doom2\\doomw.wad");
+      defs.put("iwad",     "/usr/share/games/doom/doom2.wad");
       defs.put("twad1",    "");
       defs.put("twad2",    "");
       defs.put("twad3",    "");
